styles.css

```
@@ -2,4 +2,50 @@ body {
     background-color: lightseagreen;
 }
 
+footer {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 100%;
+    justify-content: center;
+    height: 500px;
+}
+
+footer .footer-container {
+    display: flex;
+    flex-direction: column;
+    width: 700px;
+    border: 2px solid red; 
+    justify-content: center;
+}
+
+footer .footer-icons {
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+    justify-content: space-between;
+}
+
+footer .footer-madeby {
+    display: flex;
+    flex-direction: column;
+    width: 50%;
+    justify-content: space-evenly;
+    align-items: center;
```

This can be simplified a bit

```suggestion
    align-items: center;
```